include/zephyr/bluetooth/bluetooth.h

```diff
@@ -1569,6 +1569,22 @@ enum {
 
 	/** Only sync to packets with constant tone extension */
 	BT_LE_PER_ADV_SYNC_TRANSFER_OPT_SYNC_ONLY_CTE = BIT(3),
+
+	/**
+	 * @brief Sync to received PAST packets but don't generate sync reports
+	 *
+	 * This option must not be set at the same time as
+	 * @ref BT_LE_PER_ADV_SYNC_TRANSFER_OPT_FILTER_DUPLICATES .
```

Comment:
```suggestion
	 * @ref BT_LE_PER_ADV_SYNC_TRANSFER_OPT_FILTER_DUPLICATES.
```

Reply:
I wasn't sure if doxygen would be ok with a full stop running into the variable name, that's why i left the space. I can remove  if you want :)

Reply:
It works :) See e.g. https://docs.zephyrproject.org/latest/connectivity/bluetooth/api/gap.html#c.@40.BT_LE_ADV_OPT_NO_2M that has `@ref BT_LE_ADV_OPT_EXT_ADV.`